app/src/main/java/com/architectcoders/arquitectomarvel/data/database/CharacterEntity.kt

```diff
@@ -17,7 +20,7 @@ data class CharacterEntity(
     val comicListAvailable: Int?,
     val pageNumber: Int?,
     val insertDate: Long?
-)
+): Parcelable
```

Comment:
@joseantoniocampillo I wouldn't reuse the DB model as UI model because we are mixing data storage and transfer concerns, and tie UI models with Room. To do that, we should create another UI model. Check this out https://stackoverflow.com/a/56058763/2713403

Reply:
👍